src/flair/flair_diffExp.py

```diff
@@ -1,15 +1,383 @@
-#! /usr/bin/env python3
+#!/usr/bin/env python3
 
+########################################################################
+# File: diaFLAIR.py
+#  executable: diaFLAIR.py
+# Purpose: wrapper for Differential Isoform Analyses
+#
+#
+# Author: Cameron M. Soulette
+# History:      cms 01/17/2019 Created
+#
+########################################################################
+
+
+########################################################################
+# Hot Imports & Global Variable
+########################################################################
+
+
+import os
 import sys
 import argparse
 import subprocess
-import os
-from flair import set_unix_path, check_diffexp_dependencies
-os.environ['OPENBLAS_NUM_THREADS'] = '1'
+import codecs
+import errno
+from collections import Counter
+from statistics import median,stdev
+from scipy.stats import ttest_ind, ranksums
+from statsmodels.stats.multitest import multipletests
+from statistics import median, mean
+
+from flair import check_diffexp_dependencies, set_unix_path
 
 check_diffexp_dependencies()
 
-# TODO: this is a complex interface to runDE, merge the two
+os.environ['OPENBLAS_NUM_THREADS'] = '1'
+import numpy as np
+import pandas as pd
+
+
+
+scriptPath = os.path.realpath(__file__)
+path = "/".join(scriptPath.split("/")[:-1])
+runDE = path + "/" + "runDE.py"
+runDU = path + "/" + "runDU.py"
+
+
+########################################################################
+# Isoform
+########################################################################
+
+
+class Isoform(object):
+    '''
+    Object to handle isoform related data.
+
+    attributes:
+
+    methods:
+
+    '''
+
+    def __init__(self, tid=None, gid=None):
+        self.name = tid
+        self.parent = gid
+
+        self.exp  = None
+#        self.usage   = None
```

Comment:
please delete dead code, don't leave in comment out code with no reasons why. I have no idea what his means.